jib-cli/src/main/java/com/google/cloud/tools/jib/cli/jar/JarProcessors.java

```diff
@@ -17,20 +17,23 @@
 package com.google.cloud.tools.jib.cli.jar;
 
 import com.google.cloud.tools.jib.filesystem.TempDirectoryProvider;
+import com.google.common.annotations.VisibleForTesting;
 import java.io.DataInputStream;
 import java.io.IOException;
 import java.net.URL;
 import java.net.URLClassLoader;
 import java.nio.file.Path;
-import java.util.Enumeration;
+import java.util.List;
 import java.util.jar.JarEntry;
 import java.util.jar.JarFile;
+import java.util.stream.Collectors;
 
 /** Class to create a {@link JarProcessor} instance depending on jar type and processing mode. */
 public class JarProcessors {
   private static String SPRING_BOOT = "spring-boot";
   private static String STANDARD = "standard";
-  private static Integer VERSION_NOT_FOUND = 0;
+
+  @VisibleForTesting static Integer VERSION_NOT_FOUND = 0;
```

Comment:
I'm fine just using the number 0 in the test. Actually, the test can verify the literal (`assertThat(version).isEqualTo(0)`).